aana/deployments/whisper_deployment.py

```diff
@@ -79,18 +80,19 @@ class WhisperModelSize(str, Enum):
     LARGE_V2 = "large-v2"
     LARGE_V3 = "large-v3"
     LARGE = "large"
+    TURBO = "mobiuslabsgmbh/faster-whisper-large-v3-turbo"
```

Comment:
Why does it use model from our repo instead of one from original faster-whisper? I would be good to mention it in the comment right here.

Reply:
Should we call it LARGE_V3_TURBO in case there will be other turbo models later?

Reply:
Would it also work with official faster whisper or do we have to always use our fork now?

Reply:
1. SYSTRAN/faster-whisper do not have turbo model in HF. -Added in comments
2. I really don't think so, since they took 1 year to update only this. There is no such announcement/roadmap. In case some models come, we will have to test them first and name them MEDIUM_TURBO, LARGE_V2_TURBO etc and this will stay as the default one. 
3. This is with official faster whisper, we are not using our fork yet.